### PR TITLE
Pro 2954 hcaptcha a3

### PR DESCRIPTION
**Pull request by haroun**
## Summary

Enable hCaptcha for apostrophe login

## What are the specific steps to test this change?

1. Enable `@apostrophecms/login-hcaptcha` module in `app.js`
2. Use the following options in `@apostrophecms/login` module for testing
```js
  '@apostrophecms/login': {
    options: {
      hcaptcha: {
        site: '10000000-ffff-ffff-ffff-000000000001',
        secret: '0x0000000000000000000000000000000000000000'
      }
    }
  }
```
3. Validate the captcha if needed and login 

## What kind of change does this PR introduce?
*(Check at least one)*

- [ ] Bug fix
- [x] New feature
- [ ] Refactor
- [ ] Documentation
- [ ] Build-related changes
- [ ] Other

## Make sure the PR fulfills these requirements:

- [x] It includes a) the existing issue ID being resolved, b) a convincing reason for adding this feature, or c) a clear description of the bug it resolves
- [ ] The changelog is updated
- [ ] Related documentation has been updated
- [ ] Related tests have been updated

If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -38,16 +36,20 @@ module.exports = {
 
         try {
           const url = 'https://hcaptcha.com/siteverify';
-          const hcaptchaUri = `${url}?secret=${secret}&response=${token}`;
-
-          const response = await self.apos.http.post(hcaptchaUri);
+          const options = {
+            body: `response=${token}&secret=${secret}`,
```
Comment (ETLaurent) [inline index.js:40]:
```suggestion
            body: {
              response: token,
              secret
            },
```

**Review by ETLaurent — COMMENTED** [review]
image

some small `recaptcha` terms left to replace in the README

**Review by ETLaurent — APPROVED** [review]
tested

File: index.js (hunk 1 above)
Comment (haroun) [reply index.js:40]: FYI, hcaptcha requires form data, using an Object will send a JSON response

**Comment by haroun** [comment]
> image
> 
> some small `recaptcha` terms left to replace in the README

missed it, I've fixed it

**Review by boutell — APPROVED** [review]

**Pull request merged**